drm.c

```diff
@@ -649,7 +649,8 @@ int display_engine_start(int drm_fd, unsigned int width, unsigned int height,
 			 struct format_description *format,
 			 struct video_buffer *video_buffers, unsigned int count,
 			 struct gem_buffer **buffers,
-			 struct display_setup *setup)
+			 struct display_setup *setup,
+			 bool quiet)
```

Comment:
These functions are not expected to interact with the user other than for printing out errors, so I don't think I'll take this in.